x-pack/solutions/observability/plugins/serverless_observability/public/navigation_tree.ts

```diff
@@ -355,10 +355,40 @@ export const createNavigationTree = ({
         breadcrumbStatus: 'hidden',
         children: [
           {
-            link: 'management',
+            id: 'stack_management',
```

Comment:
This removes the "four squares" icon from the nav item and creates a single click target, which has an icon of an arrow pointing to the right.